### ISSUE TITLE
Pr5: en `Set2`, `setToList` es incorrecta

### ISSUE THREAD

**Issue by fidel-ml** [NahuelMGithub/Estructura-de-Datos#29]
Básicamente, estás retornando repetidos, pero si estás transformando un set a lista, no debería haberlos...

**Comment by NahuelMGithub** [comment]
claro, porque en s[i devuelve una lista, pero no una lista que sea de un Set,  es decir, sin repetidos.

**Comment by fidel-ml** [comment]
Al rehacer sinRepetidos, VOLVÉS a cometer EL MISMO error que en TODAS las prácticas anteriores... :(
¿Todavía no le creés a la hermosa RE con subtareas? :( :(
